tests/test_communication_manager.py

```diff
@@ -250,7 +250,7 @@ def test_communication_manager_can_inform_new_job_to_the_optimizer():
     )
     optimizer_job = communication_manager.optimizer.job
     assert optimizer_job.weights == true_job.weights
-    assert optimizer_job.job_type == true_job.job_type
+    # assert optimizer_job.job_type == true_job.job_type
```

Comment:
Why was this line commented out? Was it intentional?

Reply:
Yes, it's intentional. We shouldn't be checking anything based on the input jobtype as that shouldn't matter.

Reply:
Okay then I think you should just go ahead and delete this line.

Reply:
Done